wordpress-mobile/WordPress-iOS
# [Domain Management] All Domains Navigation Button

Refs #21777 

# Testing Steps
1. Install & Login Jetpack
2. Enable Domain Management Remote Feature Flag
3. MySite -> More -> Domains -> All Domains
4. Verify if the all domains screen with the search bar appears.
5. Me -> All Domains
6. Verify (for regression) if the same screen appears with the same UI.

## Regression Notes
1. Potential unintended areas of impact
All Domains screen in Me

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
